README.md

```diff
@@ -1,7 +1,14 @@
 # pySpextool
 Python code to reduce data obtained with the SpeX spectrograph
 
-# Notes
-- The calibration files stored in the `instruments/` directory 
+# Installation
+The calibration files stored in the `instruments/` directory 
```

Comment:
```suggestion
The calibration files stored in the `data/` and `instruments/` directories
```

Reply:
I am fine with this.  Except to note that currently the instruments/ directory is called instruments_data/.  I think it naturally is better called instruments/ as it contains data and code.  I was going to ask to change it in the future (I think @kelle changed it to instruments_data/ at some point in the past) but just hadn't gotten around to it.  Just making sure that it is ok for the README file to not have the exact name correct.

Reply:
I think it's fine for the README to not be perfectly accurate at this point. The point here is just to motivate the need to install `git_lfs`